### PR TITLE
Define Fujitsu Scrum Ceremonies, incorporate other PRs into one directory

### PR DESCRIPTION
**Pull request by KirstieJane**
I (@KirstieJane) am currently working on a COVID19 related project which means I'm slow to review PRs for this project.

There are quite a few issues and PRs open where I'm the blocker and many of them I have small formatting change requests.

Rather than leave those reviews and potentially hold up the process again, I'm going to pull those changes into this PR and hopefully have them all be closed by this one pull request.

I've incorporated the following PRs and issues into one pull request focused on defining the Fujitsu Scrum meetings / ceremonies and indicating where describing the process for where agendas & minutes should be stored:

- [x] Issue #143, PR #144 - Create Template.md - combined with PR #123 - Create FujitsuMeetingDates
  * Reformatted and renamed notes from call on 8 April
  * Created README file describing the Fujitsu Scrum Ceremonies - includes some adaption of the text - please check
  * Adapted meeting agenda / minutes templates - please review
  - [x] **Question**: It looks from the agenda proposed in #144 that the backlog refinement meeting and sprint planning meeting are internal to Fujitsu - is that right? I thought Turing folks also attended those meetings?
  - [x] **Question**: In the sprint planning meeting the first agenda item is "Confirm with PO there have been no changes to the backlog priority"... does that mean since the last backlog refinement meeting? 
  - [x] **Question**: What happens in a sprint planning meeting if the previous sprint is _not_ closed? (2nd item on the agenda)
  - [x] **Question**: What does "story point" all issues mean?
  * Added table of upcoming and past meetings to the README file - please review
    * Specifically - did the meetings in March happen? Do we have any minutes from them?
- [x] PR #126 - [Ready for review] Upload notes from Sprint 2 backlog refinement meeting 31-03-2020
  * Added minutes to new `project-management/fujitsu-scrum-ceremonies/agendas-and-minutes` folder

### Other PRs not addressed in this PR

- [x] PR #142 - Update agile-open-source-workflow.md -- merged already - easy to reivew ✔️ 
- [x] PR #141 - Create Sprint_1.md -- this PR is not ready for reivew - I've left some comments ❌ 

## How to review this pull request

This is a difficult PR to review because so much of what I've done is reformatting the files and that's what GitHub pays lots of attention to. You can imagine that if I "reformatted"  a bunch of code files I would potentially be completely changing what the instructions to the computer were saying!

I'd recommend going to the following links and reading the files in the `repo-tidy-20200412` branch as they are rendered there. If they capture everything in the previous pull requests than you can approve this PR.

* New folder on fujitsu scrum ceremonies: https://github.com/alan-turing-institute/AutisticaCitizenScience/tree/repo-tidy-20200412/project-management/fujitsu-scrum-ceremonies
  * Inside that folder is a README file - please read carefully and add comments to adapt the language if needed. Tagging @SaimaButt for approval.
  * Inside that folder are template agendas - please look inside each of those and check that they make sense. The use of the templates is defined in the README - please review and add any questions as comments if you don't understand. This will primarily affect @driscolle I think.
  * Inside that folder is another folder called [agendas-and-minutes](https://github.com/alan-turing-institute/AutisticaCitizenScience/tree/repo-tidy-20200412/project-management/fujitsu-scrum-ceremonies/agendas-and-minutes) - please check that these files capture future plans or recent minutes appropriately

### CODE REVIEW

**Comment by driscolle** [comment]
Hi Kirstie, I have asked Saima to respond to the questions on Github accordingly.

Regards,

Emilia Driscoll
Project Manager
MIS - Solution Implementation Directorate
Fujitsu
Mob: +44 (0) 7867821962 or Internally 73 821962
Email: Emilia.Driscoll@uk.fujitsu.com<mailto:firstname.surname@xx.fujitsu.com>
Web: uk.fujitsu.com<http://uk.fujitsu.com/>
[youtube-icon.gif]<http://www.youtube.com/user/fujitsuUK> [Facebook-icon.gif] <http://www.facebook.com/fujitsuuk>  [twitter-icon.gif] <http://twitter.com/#!/fujitsu_uk>  [linkedin-icon.gif] <http://www.linkedin.com/company/fujitsu-uk-and-ireland>  [blogger.png] <http://blog.uk.fujitsu.com/>  [google-plus-icon.gif] <https://plus.google.com/103287532874520008913/>

**Comment by EllenDevereux** [comment]
Question: It looks from the agenda proposed in #144 that the backlog refinement meeting and sprint planning meeting are internal to Fujitsu - is that right? I thought Turing folks also attended those meetings? 
Alan Turing are invited to these meetings, we will update the file to reflect that. We view Alan Turing as the product owner.

**Comment by EllenDevereux** [comment]
Question: In the sprint planning meeting the first agenda item is "Confirm with PO there have been no changes to the backlog priority"... does that mean since the last backlog refinement meeting?

Yes it does, we will add this clarity to the document

**Comment by EllenDevereux** [comment]
Question: What happens in a sprint planning meeting if the previous sprint is not closed? (2nd item on the agenda)

The sprint is always closed. Any outstanding issues are reprioritised and moved to the next sprint.

**Comment by EllenDevereux** [comment]
Question: What does "story point" all issues mean?

story point is a metric used in Agile project development to estimate the difficulty of implementing a given issue (story=usual agile terminology)

elements considered in assigning story points include the complexity of the issue, any unknown factors and the potential effort required to implement it. 

This clarification will be added to the document. We will double check whether this is in the Agile methodology document and add it if not.

**Comment by EllenDevereux** [comment]
Hi @KirstieJane I have made a couple of small changes to the readme file to tidy up a couple of points, all of which were covered in the comments in this thread. I've added the minutes for the sprint 1 retrospective also.

File: project-management/fujitsu-scrum-ceremonies/agendas-and-minutes/20200401_sprint01_sprint-retrospective.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+* Thomas Stanley
+* Christippher Legg
+* Emilia Driscoll
+* Ellen Devereux
+* Harrison Cannon
+* Conor Henderson
+* Joe Collins
+* Jake Wych
+* Ryan Gallery
+* Daniel Ball
```
Comment (KirstieJane) [inline project-management/fujitsu-scrum-ceremonies/agendas-and-minutes/20200401_sprint01_sprint-retrospective.md:14]: Fine for now but I recommend first name and last initial to maintain just a little bit of privacy in these public minutes.

File: project-management/fujitsu-scrum-ceremonies/agendas-and-minutes/20200401_sprint01_sprint-retrospective.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+### Challenges
+*	Understanding GitHub
+* Slow development of Demo and Report
+*	Communication between Autistica and Fujitsu
+* Some devs not being around for all meetings
+*	High Expectations
```
Comment (KirstieJane) [inline project-management/fujitsu-scrum-ceremonies/agendas-and-minutes/20200401_sprint01_sprint-retrospective.md:21]:
```suggestion
* Understanding GitHub
* Slow development of Demo and Report
* Communication between Autistica and Fujitsu
* Some devs not being around for all meetings
* High Expectations

```

File: project-management/fujitsu-scrum-ceremonies/agendas-and-minutes/20200401_sprint01_sprint-retrospective.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+### What went well
+*	Highlighting issues and identifying plans and processes
+*	Fridays demo meeting went well with good feedback
+*	Everyone is supportive and helpful
+*	Openess within the team to be able to raise issues
```
Comment (KirstieJane) [inline project-management/fujitsu-scrum-ceremonies/agendas-and-minutes/20200401_sprint01_sprint-retrospective.md:26]:
```suggestion
* Highlighting issues and identifying plans and processes
* Fridays demo meeting went well with good feedback
* Everyone is supportive and helpful
* Openess within the team to be able to raise issues

```

File: project-management/fujitsu-scrum-ceremonies/template-backlog-refinement.md [hunk 4]
```diff
@@ -3,7 +3,7 @@
 ## Attendees
 
 * Scrum Master: *Name of attendee*
-* Product Owner: *Name of attendee*
+* Product Owner (Alan Turing): *Name of attendee*
```
Comment (KirstieJane) [inline project-management/fujitsu-scrum-ceremonies/template-backlog-refinement.md:6]:
```suggestion
* Product Owner (Turing Institute): *Name of attendee*
```

File: project-management/fujitsu-scrum-ceremonies/template-sprint-planning.md [hunk 5]
```diff
@@ -3,18 +3,18 @@
 ## Attendees
 
 * Scrum Master: *Name of attendee*
-* Product Owner: *Name of attendee*
+* Product Owner (Alan Turing): *Name of attendee*
```
Comment (KirstieJane) [inline project-management/fujitsu-scrum-ceremonies/template-sprint-planning.md:6]:
```suggestion
* Product Owner (Turing Institute): *Name of attendee*
```

File: project-management/fujitsu-scrum-ceremonies/template-sprint-planning.md [hunk 6]
```diff
@@ -3,18 +3,18 @@
...
 
 ## Agenda
 
-1. [Confirm backlog priority](#1-confirm-backlog-priority): Confirm with PO there have been no changes to the backlog priority
-2. [Confirm previous sprint is closed](#2-confirm-previous-sprint-is-closed)
+1. [Confirm backlog priority](#1-confirm-backlog-priority): Confirm with PO there have been no changes to the backlog priority since the last backlog refinement meeting.
+2. [Sprint Retrospective notes](#2-sprint-retrospective-notes): Consider any changes that need to be made to the sprint process based on the sprint retrospective
 3. [Team availability](#3-team-availability): Capture team member's availability for the upcoming sprint
-4. [Story point issues in sprint](#4-story-point-issues-in-sprint)
+4. [Story point issues in sprint](#4-story-point-issues-in-sprint): "Story Point" is a metric used in Agile project development to estimate the difficulty of implementing a given issue.
```
Comment (KirstieJane) [inline project-management/fujitsu-scrum-ceremonies/template-sprint-planning.md]: This is great! Thank you!

File: project-management/fujitsu-scrum-ceremonies/template-sprint-retrospective.md [hunk 7]
```diff
@@ -3,7 +3,7 @@
 ## Attendees
 
 * Scrum Master: *Name of attendee*
-* Product Owner: *Name of attendee*
+* Product Owner (Alan Turing): *Name of attendee*
```
Comment (KirstieJane) [inline project-management/fujitsu-scrum-ceremonies/template-sprint-retrospective.md:6]:
```suggestion
* Product Owner (Turing Institute): *Name of attendee*
```

File: project-management/fujitsu-scrum-ceremonies/agendas-and-minutes/20200401_sprint01_sprint-retrospective.md (hunk 1 above)
Comment (EllenDevereux) [reply project-management/fujitsu-scrum-ceremonies/agendas-and-minutes/20200401_sprint01_sprint-retrospective.md:14]: Sure

**Comment by EllenDevereux** [comment]
I'm happy with this, thanks for organising these files better! I'm happy for it to be merged. @KirstieJane.

**Pull request merged**